packages/web/app/src/lib/preflight-sandbox/graphiql-plugin.tsx

```diff
@@ -299,6 +332,16 @@ export function usePreflightScript(args: {
           setLogs(logs => [...logs, log]);
           return;
         }
+
+        if (ev.data.type === IFrameEvents.Outgoing.Event.ready) {
+          return;
+        }
+
+        if (ev.data.type === IFrameEvents.Outgoing.Event.start) {
```

Comment:
I like the explicitness of handling all cases. Is it worth adding debug logging here in case e.g. start is called but not ready? I'm not sure what the exact flow is or what could go wrong.

Reply:
When you say debug logging, are we talking console.log or something more specialized?

Reply:
Yeah like `console.debug` or send an alert to sentry if we're worried about a specific case.